apps/gp2-server/src/utils/odata.ts

```diff
@@ -0,0 +1,24 @@
+import buildQuery, { Filter } from 'odata-query';
+
+const FILTER_URL_QUERY_PARAM = '?$filter=';
+
+export const buildODataFilter = (filterObj: Filter): string =>
+  buildQuery({ filter: filterObj }).replace(FILTER_URL_QUERY_PARAM, '');
+
+export const buildEqFilterForWords = (
```

Comment:
where is this used?

Reply:
if it's the same as crn we can move it to a shared package

Reply:
it's a util package copied over from CRN.

Once we move to Contentful, it should be deleted.

Reply:
but are we using the second function?